src/test/java/redis/clients/jedis/JedisPooledTest.java

```diff
@@ -21,12 +21,12 @@
 
 public class JedisPooledTest {
 
-  private static final HostAndPort hnp = HostAndPorts.getRedisServers().get(7);
-  private static final HostAndPort pwp = HostAndPorts.getRedisServers().get(1); // password protected
+  private static final EndpointConfig endpoint1 = HostAndPorts.getRedisEndpoint("standalone7-with-lfu-policy");
```

Comment:
Same here, I would add some semantics to the variable names, not just `endpointX`.

Reply:
Fixed